File: lecture0-30/lecture003/pascal.cpp
#include<iostream>

using namespace std;
int val_(int n){
    //int n;
    for(int i=0;i<n;i++){
        int val=1;
        for(int j=0;j<=i;j++){
            cout<<val;
            val=(i-j)*val/(j+1);
        }
    cout<<endl;
    
    }
}
int main(int args,char**argv){
    int n;
    //int val;
    cout<<"Enter no of rows : ";
    cin>>n;
    val_(n);
    
}

File: lecture003/pascal.java
import java.util.Scanner;

public class pascal {
    public static Scanner scn = new Scanner(System.in);

    public static void main(String[] args) {

        System.out.print("Enter no of rows : ");
        int n;
        n = scn.nextInt();
        val_(n);

    }

    public static void val_(int n) {
        for (int i = 0; i < n; i++) {
            int val = 1;
            for (int j = 0; j <= i; j++) {
                System.out.print(val);
                val = (i - j) * val / (j + 1);
            }
            System.out.println();
        }
    }
}